### PR TITLE
Docs: Add info and links about our hosted services to our grafana install docs

### PR DESCRIPTION
**Pull request by torkelo**
* [x] Added to debian install 
* [x] Added to rpm install 
* [x] Added to docker install
* [x] Added to prometheus install
* [x] Added to loki install

### CODE REVIEW

**Comment by osg-grafana** [comment]
@Eve832 is lead on Cloud.

File: docs/sources/installation/rpm.md [hunk 1]
```diff
@@ -22,10 +26,12 @@ If you install from the YUM repository, then Grafana is automatically updated ev
...
 | ------------------------- | ------------------ | -------------------------------------------------- |
-| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/rpm`             |
-| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/rpm-beta`        |
 | Grafana Enterprise        | grafana-enterprise | `https://packages.grafana.com/enterprise/rpm`      |
 | Grafana Enterprise (Beta) | grafana-enterprise | `https://packages.grafana.com/enterprise/rpm-beta` |
+| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/rpm`             |
+| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/rpm-beta`        |
+
+> The Enterprise Edition is the default and recommended edition. It includes all the features of the OSS Edition, can be used for free and can be upgraded to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), including support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```
Comment (achatterjee-grafana) [inline docs/sources/installation/rpm.md:34]:
```suggestion
>**Note:** The Enterprise is the recommended and default option. It is available for free and includes all the features of the OSS Edition. You can also upgrade to the [ full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page) and has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```

File: docs/sources/datasources/prometheus.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ weight = 1300
 
 Grafana includes built-in support for Prometheus. This topic explains options, variables, querying, and other options specific to the Prometheus data source. Refer to [Add a data source]({{< relref "add-a-data-source.md" >}}) for instructions on how to add a data source to Grafana. Only users with the organization admin role can add data sources.
 
+## Hosted Prometheus
+
+You can run Prometheus on your own hardware or use [Grafana Cloud](https://grafana.com/products/cloud/features/#cloud-dashboards-grafana). The free forever plan includes Grafana, 10K Prometheus series, 50 GB logs and more. [Create a free account to get started](https://grafana.com/auth/sign-up/create-user?pg=doc-grafana-&plcmt=in-text).
+
```
Comment (achatterjee-grafana) [inline docs/sources/datasources/prometheus.md:16]:
```suggestion
Run Grafana on premise, using your own infrastructure or on [Grafana Cloud](https://grafana.com/products/cloud/features/#cloud-dashboards-grafana) and get Grafana without the overhead of installing, maintaining, and scaling your observability stack. The free forever plan includes Grafana, 10K Prometheus series, 50 GB logs and more. [Create a free account](https://grafana.com/auth/sign-up/create-user?pg=docs-grafana-docker&plcmt=in-text) to get started.

```

File: docs/sources/installation/debian.md [hunk 3]
```diff
@@ -22,12 +26,12 @@ If you install from the APT repository, then Grafana is automatically updated ev
...
-| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/deb beta main`          |
 | Grafana Enterprise        | grafana-enterprise | `https://packages.grafana.com/enterprise/deb stable main` |
 | Grafana Enterprise (Beta) | grafana-enterprise | `https://packages.grafana.com/enterprise/deb beta main`   |
+| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/deb stable main`        |
+| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/deb beta main`          |
 
-> We recommend all users install the Enterprise Edition of Grafana, which can be seamlessly upgraded with a Grafana Enterprise [subscription](https://grafana.com/products/enterprise/?utm_source=grafana-install-page).
+> The Enterprise Edition is the default and recommended edition. It includes all the features of the OSS Edition, can be used for free and can be upgraded to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), including support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```
Comment (achatterjee-grafana) [inline docs/sources/installation/debian.md:34]:
```suggestion
>**Note:** The Enterprise is the recommended and default option. It is available for free and includes all the features of the OSS Edition. You can also upgrade to the [ full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page) and has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```

Comment (achatterjee-grafana) [reply docs/sources/installation/debian.md:34]: In the table above, I see only one Enterprise offering. But from the note, it seems like there are two enterprise offerings, one free and one licensed. Do we need to update the table? Thank you.

File: docs/sources/datasources/prometheus.md (hunk 2 above)
Comment (achatterjee-grafana) [reply docs/sources/datasources/prometheus.md:16]: Similar changes are needed in other topics.

**Review by achatterjee-grafana — COMMENTED** [review]
Added doc review.

File: docs/sources/installation/debian.md (hunk 3 above)
Comment (achatterjee-grafana) [reply docs/sources/installation/debian.md:34]: Should this be free, pro, and advanced Cloud tiers instead?

**Comment by osg-grafana** [comment]
cc @mjseaman thoughts on this verbiage?

**Review by KMiller-Grafana — CHANGES_REQUESTED** [review]
I've looked only at the Loki datasource docs changes within this PR.

**Pull request merged**